main/src/cgeo/geocaching/EditWaypointActivity.java

```diff
@@ -382,6 +390,7 @@ public void updateGeoData(final GeoData geo) {
         public void run() {
             try {
                 waypoint = DataStore.loadWaypoint(waypointId);
+                calcState = waypoint.getCalculatorSaveState();
```

Comment:
It was called "get...SavedState" in particular as it is retrieving the previously saved state rather than the current state.  I'll change this to "getCalculatorStoredState(...)" if that's better.